roles-in-tech/technical-writing.md

```diff
@@ -1,90 +1,67 @@
-# Example Topic Outline (Replace with Title)
+# Technical Writing
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+About 30 minutes
 
 ### Prerequisites
 
+[Reading & Writing Documentation](https://github.com/Techtonica/curriculum/blob/master/reading-and-writing-documentation/documentation.md)
 Here are links to lessons that should be completed before this lesson:
```

Comment:
Please replace line 12 with the reading & writing documentation link, unless you have a reason to require the Git lesson.